solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -1324,6 +1335,16 @@ public static SolrQueryRequest req(SolrParams params, String... moreParams) {
     return new LocalSolrQueryRequest(h.getCore(), mp);
   }
 
+  /** Generates a SolrQueryRequest randomly assigning multi or single threaded processing. */
+  public static SolrQueryRequest reqRandMulti(SolrParams params, String... moreParams) {
```

Comment:
I don't like this approach.  Imagine that we embrace this -- would this then mean all/most calls to `req` would switch to `reqAndMulti`?  Not good.  Assuming we imagine that `multiThreaded=true` should just work (even if may not actually use multiple threads for various reasons), I'd prefer we change a system property to have true/false and randomly chosen.